pandas/tests/indexes/test_base.py

```diff
@@ -565,7 +575,8 @@ def test_delete(self, pos, expected):
 
     def test_delete_raises(self):
         index = Index(['a', 'b', 'c', 'd'], name='index')
-        with pytest.raises((IndexError, ValueError)):
+        msg = "index 5 is out of bounds for axis 0 with size 4"
+        with pytest.raises((IndexError, ValueError), match=msg):
             # either depending on numpy version
```

Comment:
I suppose unrelated to this PR but do we know which numpy version this differs in? Wondering if our bumps make this irrelevant

Reply:
i'll change to `IndexError` only and see what breaks.